src/tls/boring.rs

```diff
@@ -569,6 +569,7 @@ where
 
     fn accept(&self, conn: TcpStream) -> Self::AcceptFuture {
         let mut acceptor = self.acceptor.clone();
+        conn.set_nodelay(true).unwrap();
```

Comment:
you can do `SslStream.get_ref` to get the inner TcpStream. Then you can call set_nodelay.

So I think this is probably not the best place.

Maybe in http_util::tls_server add

```
        .map(|conn| {
            conn.get_ref().set_nodelay()
        })
```

?

Reply:
> you can do `SslStream.get_ref` to get the inner TcpStream. Then you can call set_nodelay.
> 
> So I think this is probably not the best place.
> 
> Maybe in http_util::tls_server add
> 
> ```
>         .map(|conn| {
>             conn.get_ref().set_nodelay()
>         })
> ```
> 
> ?

@howardjohn NGL I'm a bit surprised this works. I thought that setting NODELAY after the TLS handshake would cause handshakes to be really slow, but that doesn't seem to be the case (at least locally).

Reply:
oh I see the problem, its after the TLS handshake. I didn't think of that. I guess current way is fine then.. I don't mind either way I guess